### DISCUSSION TITLE
Move push notifications to different monitor (Win 11)

### DISCUSSION THREAD

**Discussion by cengels** [ramensoftware/windhawk-mods#504] [Mod ideas]
One thing that's always bothered me about Windows 10 and 11 is that push notifications always appear on the primary monitor, even if you move the primary taskbar to another monitor (which you can't even do anymore in Windows 11).

The main problem with this is full-screen applications like games. Push notifications won't appear on top of full-screen applications, and full-screen applications always launch on the primary monitor as well, so I'll miss all notifications while a full-screen app is maximized.

In theory, notifications *should* still appear on top of full-screen applications unless a corresponding do-not-disturb rule in the Windows settings is enabled, but that hasn't personally been the case for me with a lot of apps and their notifications.

On that note, I wonder if it would be possible to add the notification tray to non-primary taskbars as well.

I realize this may be tricky to do, but it would be incredibly awesome if it could be done.

**Comment by m417z** [comment]
Hi, I just created a mod that allows to have notifications displayed on a different monitor:
https://windhawk.net/mods/notifications-placement

With your use case of fullscreen games, I'm not sure that it will be enough, though, as I suspect Windows just doesn't show notifications when in fullscreen mode.

**Reply by AJolly** [reply]
Is it possible to clone notifications - display them on all monitors?  I often find that I don't catch the notifications w 3 or 4 monitors.

**Reply by m417z** [reply]
Currently that's not possible.

**Reply by KinqWolf** [reply]
> Merhaba, bildirimlerin farklı bir monitörde görüntülenmesini sağlayan bir mod oluşturdum: https://windhawk.net/mods/notifications-placement
> 
> Tam ekran oyunlarda kullanım durumunuz göz önüne alındığında bunun yeterli olacağından emin değilim, çünkü Windows'un tam ekran modundayken bildirimleri göstermediğinden şüpheleniyorum.

İşime yaradı çok teşekkürler.

**Comment by higgs312** [comment]
Works great except for Teams notifications. They still appear on the main monitor. I tried adding the path for the Teams executable to the "Custom process inclusion list" but it didn't work.
Any way to get this to work?

**Reply by m417z** [reply]
Do you have a screenshot? Perhaps Teams implements custom notifications.

**Reply by m417z** [reply]
Looks like indeed they implement custom notifications, and there seems to be an option to use the native Windows notifications, try setting this option:
https://support.microsoft.com/en-us/office/change-your-notifications-style-in-microsoft-teams-0da93820-93d0-4da3-86b6-fc28d19908e3

**Reply by higgs312** [reply]
I hope that this is what you are looking for
![Capture](https://github.com/user-attachments/assets/f3427883-48c5-428f-8c56-b5439994eba7)

**Reply by higgs312** [reply]
I'll give it a shot...thx!!
And thx for the quick response!!

**Reply by m417z** [reply]
You're welcome! Regarding the process lists, they control which programs the mod affects. In this case, the mod affects only system processes which control and show the notifications. Including other processes doesn't make sense for this mod, as other processes don't show notifications directly - they signal the system which shows the notifications in a centralized manner.
